apps/src/blockly/customBlocks/googleBlockly/spritelabBlocks.js

```diff
@@ -55,7 +53,6 @@ export const blocks = {
       if (!block.getInput(INPUTS.FLYOUT)) {
         const flyoutField = createFlyoutField(block);
         flyoutField.showEditor();
-        flyoutField.render_();
```

Comment:
Is this part of the flyout fields clean-up?

Reply:
Yes! It was actually just redundant. Normally `render_` should only be called by `getSize`.